ert/data/record/_record.py

```diff
@@ -59,19 +59,49 @@ class RecordType(str, Enum):
 
 
 class Record(ABC):
+    """The :py:class:`Record` class is an abstract class that
+    includes record data and the record type.
+
+    The record needs be either `BlobRecord` or `NumericalRecord` type derived
+    from `Record` class
+    """
+
     @property
     @abstractmethod
     def data(self) -> record_data:
+        """Returns record data.
+
+        :return: Data is of Union[numerical_record_data, blob_record_data]
+        :rtype: record_data
```

Comment:
There exists plugins to sphinx that can get this `rtype` from the typing information in line 71. Has that been considered?